sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -18,12 +18,15 @@
         @click="wizardGoPrevious"
         >Previous section</v-btn
       >
+      <v-btn color="primary" v-show="isFirstPage" @click="wizardGoNext"
```

Comment:
The navigation buttons can be probably replaced by the `<footer-button>` component with the `justify="space-between"`.
The navigation button on the right using the `@click="wizardGoNext"` can be probably the same with the label defined by the current page.

Reply:
Thanks, I used footer-buttons.